pytest_zigzag/__init__.py

```diff
@@ -1,15 +1,17 @@
 # -*- coding: utf-8 -*-
 
-__version__ = '0.0.0'
-
 # ======================================================================================================================
 # Imports
 # ======================================================================================================================
+from __future__ import absolute_import
 import os
 import pytest
-import pkg_resources
 from datetime import datetime
+from zigzag.zigzag import ZigZag
```

Comment:
Nitpick: PyCharm doesn't like this import because of reasons. Would you mind add the `# noinspection PyPackageRequirements` directive above this statement to shut it up?